Orange/widgets/data/owpaintdata.py

```diff
@@ -1029,7 +1037,6 @@ def _check_and_set_data(data):
         else:
             self.input_data = np.column_stack((X, y))
         self.reset_to_input()
-        self.unconditional_commit()
```

Comment:
The widget should output the data (when opening a saved workflow) even if the `Send Automatically` is off. That is the point of `self.unconditional_commit` and should be kept.